src/components/register/register.test.js

```diff
@@ -56,4 +59,54 @@ describe('Register', () => {
       passphrase: 'sample passphrase',
     });
   });
+
+  it('should call activePeerSet with testnet if network index is testnet', () => {
+    const props = wrapper.find('Passphrase').props();
+
+    const loginData = stub(Utils, 'getLoginData');
```

Comment:
Please move the stub creation to `beforeEach` block and the stub `restore` to `afterEach` block. It avoids some code duplication and makes tests more stable because `restore` gets called even if there is some error in the middle of the test and no other test is affected. WIth `restore` inside the test case, in case of error in the test, the stub is not restored and the next test fails too because it is trying to stub something that already is a stub.